mappings/net/minecraft/network/NetworkSide.mapping

```diff
@@ -1,2 +1,4 @@
 CLASS net/minecraft/unmapped/C_exeqxqdd net/minecraft/network/NetworkSide
+	FIELD f_dowoouom S2C Lnet/minecraft/unmapped/C_exeqxqdd;
```

Comment:
Might be worth adding a javadoc with its meaning (something like `Server to ("2") Client` would work)